server/services/visitService.ts

```diff
@@ -1,8 +1,12 @@
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
Could you not use the same uuid from crypto here?

Reply:
Yes - I think I'd written this code (and its tests) before realising. Going to sort here and other places in next PR to be consistent and get rid of `uuid` module.